### PR TITLE
Add first SlingShotter release

### PR DESCRIPTION
**Pull request by johnfink8**

### CODE REVIEW

File: NetKAN/SlingShotter.netkan [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "spec_version": "v1.16",
```
Comment (plague006) [inline NetKAN/SlingShotter.netkan]: Only a spec_version of `"v1.4"` is required for this metadata.

File: NetKAN/SlingShotter.netkan [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "spec_version": "v1.16",
+    "identifier": "SlingShotter",
+    "$kref": "#/ckan/github/johnfink8/KerbalSlingshotter",
+    "name"        : "Kerbal SlingShotter",
+    "author"      : [ "Overengineer1" ],
+    "abstract"    : "Display positions in the map for ships, planets, and moons at a selected time.",
+    "version"     : "1.0.0",
```
Comment (plague006) [inline NetKAN/SlingShotter.netkan]: `version` should not be hardcoded as 1) all releases of the mod on github will be tagged with the same version 2) netkan can scrape the mod's version information from the github release.

**Comment by johnfink8** [comment]
Updated with version 1.4, removed the mod version.

**Comment by plague006** [comment]
Looks good, merging with my thanks.

**Pull request merged**